src/assets/styles/app.css

```diff
@@ -1,103 +0,0 @@
-/*
- * Fonts
- */
-@import url(https://fonts.googleapis.com/css?family=Open+Sans:300italic,400italic,700italic,300,400,700);
-
-/*
- * Media
- */
-.media .img {
-  float: left;
-  margin-right: 10px;
-}
-
-/*
- * Meta
- */
-.meta {
-  display: -webkit-box;
-  display: -moz-box;
-  display: -webkit-flex;
-  display: -ms-flexbox;
-  display: box;
-  display: flex;
-  -webkit-box-orient: horizontal;
-  -moz-box-orient: horizontal;
-  -o-box-orient: horizontal;
-  -webkit-box-lines: multiple;
-  -moz-box-lines: multiple;
-  -o-box-lines: multiple;
-  -webkit-flex-flow: row wrap;
-  -ms-flex-flow: row wrap;
-  flex-flow: row wrap;
-  margin-top: 5px;
-}
-.meta .term {
-  -webkit-box-flex: 1;
-  -moz-box-flex: 1;
-  -o-box-flex: 1;
-  box-flex: 1;
-  -webkit-flex: 1 250px;
-  -ms-flex: 1 250px;
-  flex: 1 250px;
-  margin-top: 10px;
-  overflow: hidden;
-  white-space: nowrap;
-  -o-text-overflow: ellipsis;
-  text-overflow: ellipsis;
-}
-.meta .value {
-  display: inline;
-  margin-left: 5px;
-  font-weight: bold;
-}
-
-/*
- * Buttons
- */
-button, .button {
-  display: inline-block;
-  position: relative;
-  color: #fff;
-  text-align: center;
-  cursor: pointer;
-}
-button, .button {
-  background-color: #368aa2;
-}
-button:hover, .button:hover {
-  background-color: #2b7187;
-}
-a.button {
-  line-height: 17px;
-  color: #fff;
-}
-a.button:hover, a.button:active, a.button:focus {
-  color: #fff;
-}
-
-/*
- * Dragula
- */
```

Comment:
Dragula moved to https://github.com/PRX/publish.prx.org/pull/411/files#diff-b133c7c05994ecce85295092da6ef47dR1